src/coffee/directives/api/models/parent/drawing-manager-parent-model.coffee

```diff
@@ -3,6 +3,24 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
     ['uiGmapLogger', '$timeout'
       ($log, $timeout) ->
         class DrawingManagerParentModel
+          setEvents: (gObject, scope, model, ignores) ->
+            if angular.isDefined(scope.events) and scope.events? and angular.isObject(scope.events)
```

Comment:
why did you cut and paste this code, pull in the EventsHelper as a service / factory and use it directly. That way I manage the code in one spot.

Same goes for removeEvents.